ethereum_bridge/src/storage/eth_bridge_queries.rs

```diff
@@ -150,21 +151,22 @@ where
     D: storage::DB + for<'iter> storage::DBIter<'iter>,
     H: storage::StorageHasher,
 {
-    fn check_bridge_status(&self) -> EthBridgeStatus {
-        BorshDeserialize::try_from_slice(
+    fn check_bridge_status(&self) -> Option<EthBridgeStatus> {
```

Comment:
This fn was panicking with the message "The Ethereum bridge active key should be in storage" when I added a `check_bridge_status`/`is_bridge_active` call to `start_ethereum_oracle_if_necessary`, so made this fn return a `None` instead if it can't find the value in storage (and `is_bridge_active` still return `False`), is this ok?

Reply:
Erg, this is a bit annoying. I think that should be ok. I guess that key is initialized in `init_chain` which gets called *after* `Shell::new`

Reply:
I can start a validator node on a fresh net with the bridge configured at genesis (at least locally?). That status key is set in `InitChain` and then that value should be available in `Shell::new` calls in further restarts (assuming one block has been synced). I'm not sure what to check here.